bin/make-xy0-release

```diff
@@ -0,0 +1,264 @@
+  exit 1
+fi
+
+if [ "$(<.git/HEAD)" != "ref: refs/heads/master" ]; then
+  echo "${FORCEABLE} Run from master branch."
+  $FORCE || exit 1
+fi
+if ! git diff --exit-code >/dev/null; then
+  echo "${FORCEABLE} Uncommitted changes."
+  $FORCE || exit 1
+fi
+
+REQUIRED_GIT_ORIGIN="git@github.com:hhvm/packaging.git"
+GIT_ORIGIN="$(git remote get-url --push origin)"
+
+if [ "$GIT_ORIGIN" != "$REQUIRED_GIT_ORIGIN" ]; then
+  echo "${FORCEABLE} Git origin push URL is unsupported:"
+  echo "  - Current value: ${GIT_ORIGIN}"
+  echo "  - Required value: ${REQUIRED_GIT_ORIGIN}"
+  echo "  - Fix: git remote set-url --push origin ${REQUIRED_GIT_ORIGIN}"
+  $FORCE || exit 1
+fi
+
+if $DRY_RUN; then
+  echo "Dry run - exiting."
+  exit 0
+fi
+
+
+echo "Fetching any updates to packaging repo..."
+git fetch >/dev/null
+if ! git diff --exit-code origin/master..HEAD >/dev/null; then
+  echo "${FORCEABLE} Current branch is out of sync with origin/master"
+  $FORCE || exit 1
+fi
+
+SED=$(if [ "$(uname -s)" == "Darwin" ]; then echo gsed; else echo sed; fi)
+
+if ! command -v "$SED" >/dev/null; then
+  echo "'$SED' is required."
+  if [ "$SED" == "gsed" ]; then
+    echo "Try: brew install gnu-sed"
+  fi
+  exit 1
+fi
+
+function statusline() {
+  # green background, back foreground
+  tput setab 2
+  tput setaf 0
+  echo "$@"
+  # reset
+  tput sgr0
+}
+
+statusline "Creating packaging branch..."
+git checkout -b "HHVM-${VERSION}"
+statusline "Marking support for new DISTRO apt repo.."
+echo "DISTRO-${VERSION}" >> DEBIAN_REPOSITORIES
+git commit DEBIAN_REPOSITORIES -m "Adding DISTRO-${VERSION} apt repositories"
+statusline "Pushing packaging branch..."
+git push -u origin "HEAD:HHVM-${VERSION}"
+
+SCRATCHDIR="$(mktemp -dt hhvm.XXXXXXXX)"
+statusline "Working in ${SCRATCHDIR}..."
+
+cd "$SCRATCHDIR"
+
+if [ "$NIGHTLY" = "existing-tag" ]; then
+  statusline "Not cloning HHVM, already tagged."
+else
+  statusline "Cloning hhvm..."
+  git clone git@github.com:hhvm/hhvm-staging.git "hhvm-${VERSION}"
+  (
+    statusline "Tagging and pushing HHVM..."
+    cd "hhvm-$VERSION"
+    git remote add staging git@github.com:hhvm/hhvm-staging.git
+    git checkout -b "HHVM-${VERSION}"
+    git reset --hard "nightly-${NIGHTLY}"
+    "$PKGDIR/bin/hhvm-tag-and-push" "${VERSION}.0"
+  )
+fi
+
+statusline "Cloning hhvm-docker..."
+git clone git@github.com:hhvm/hhvm-docker.git
+(
+  cd hhvm-docker
+  statusline "Removing 'latest' from previous release branch tags..."
+  git checkout "HHVM-${PREVIOUS_VERSION}"
+  $SED -i '/^latest$/d' EXTRA_TAGS
+  git commit EXTRA_TAGS -m "Removing 'latest' tag"
+  git push
+  statusline "Creating branch for new release..."
+  git checkout master
+  git checkout -b "HHVM-$VERSION"
+  echo latest > EXTRA_TAGS
+  echo "${VERSION}-latest" >> EXTRA_TAGS
+  git commit EXTRA_TAGS -m "Update tags for ${VERSION} release"
+  git push -u origin "HEAD:HHVM-${VERSION}"
+)
+
+statusline "Triggering Linux builds..."
+"$PKGDIR/bin/build-on-aws" "${VERSION}.0"
+
```

Comment:
Or better, let's remove the call to `build-on-aws` and rename this file to `create-release-branch`. It would makes scripts more orthogonal.

```suggestion
statusline "Triggering Linux builds..."
"$PKGDIR/bin/build-on-aws" "${VERSION}.0"
```

Reply:
The only changes are in the doc header, leaving everything else unmodified